### PR TITLE
update tensorrt cheatsheet

### PR DESCRIPTION
**Pull request by hnyu**
Added a note about `rsample()` for tensorrt.

### CODE REVIEW

File: alf/utils/README.md [hunk 1]
```diff
@@ -72,6 +72,10 @@ If you have to use a dynamic shape, you can choose to use the CUDA backend by se
 ORT_ONNX_BACKEND_EXCLUDE_PROVIDERS=TensorrtExecutionProvider
 ```
 
+### `rsample()`
+TensorRT will report "ERROR: Network must have at least one output" when there is any `rsample()` in the eager mode code. Fortunately, for inference we can usually use the distribution's mode
```
Comment (emailweixu) [inline alf/utils/README.md]: Can `sample()` be used?

Comment (hnyu) [reply alf/utils/README.md]: > Can `sample()` be used?

verified that sample() is OK.

Comment (emailweixu) [reply alf/utils/README.md]: Update the doc to say that inference we can usually use `sample()` or the distribution's mode.

Comment (hnyu) [reply alf/utils/README.md]: > Update the doc to say that inference we can usually use `sample()` or the distribution's mode.

updated

**Review by emailweixu — APPROVED** [review]

**Pull request merged**